charts/cryostat/templates/cryostat_deployment.yaml

```diff
@@ -6,13 +6,15 @@ metadata:
   name: {{ include "cryostat.fullname" . }}
   labels:
     {{- include "cryostat.labels" . | nindent 4 }}
+    component: cryostat
 spec:
   replicas: 1
   strategy:
     type: Recreate
   selector:
     matchLabels:
       {{- include "cryostat.selectorLabels" . | nindent 6 }}
+      component: cryostat
```

Comment:
Make sense to use `app.kubernetes.io/component`?

https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/

Reply:
Btw, should this PR target base branch `separate-db-storage`?

Reply:
I didn't set it up to target that base branch since that isn't an upstream feature branch, so instead I just put it as a dependency for that one to merge into `main` first and then this one would rebase and merge on top of it.

Reply:
^ except I realized I pasted the link to #191 twice instead of properly tagging the "depends on" :-)

Reply:
Ahh okayy that makes sense now :D

Reply:
Oh, I did make that an upstream feature branch... I didn't think I had. Well anyway, this branch can just get merged into main later, since it's technically a separate fix. I don't want to make the eventual integration merge too much bigger and more complex than it already will be.